Update to the UCT overview page (#7)
AdobeDocs/commerce-operations.en: help/upgrade/upgrade-compatibility-tool/overview.md

## Help improve the [!DNL Upgrade Compatibility Tool]

To connect with the [!DNL Upgrade Compatibility Tool] team, contact us on the Engineering Slack channel [[!DNL Upgrade Compatibility Tool]](https://magentocommeng.slack.com/archives/C019Y143U9F). We want to hear your feedback, issues, and suggestions to help us improve the tool.

> The Upgrade Compatibility Tool uses rules defined within our [Coding Standards](https://devdocs.magento.com/guides/v2.4/coding-standards/bk-coding-standards.html), to ensure your project is following Adobe Commerce best practices, and at the same time allows you to improve and extend the Upgrade Compatibility Tool compatibility analysis by contributing to this project. Check the [Contribute](https://devdocs.magento.com/guides/v2.4/coding-standards/contributing.html)  page for more information.

Reviewer: ```suggestion
The [!DNL Upgrade Compatibility Tool] uses rules defined within our [Coding Standards](https://devdocs.magento.com/guides/v2.4/coding-standards/bk-coding-standards.html) to ensure that your project is following Adobe Commerce best practices and to help you improve and extend the [!DNL Upgrade Compatibility Tool]. Check the [Contributing](https://devdocs.magento.com/guides/v2.4/coding-standards/contributing.html) page for more information.
```

Reviewer 2: @jeff-matthews should be  [!DNL Upgrade Compatibility Tool] right?

Reviewer: @danidelcar, yes. My mistake. I'll edit my suggestion.

Reviewer: @danidelcar, suggestion updated.

> The Upgrade Compatibility Tool uses rules defined within our [Coding Standards](https://devdocs.magento.com/guides/v2.4/coding-standards/bk-coding-standards.html), to ensure your project is following Adobe Commerce best practices, and at the same time allows you to improve and extend the Upgrade Compatibility Tool compatibility analysis by contributing to this project. Check the [Contribute](https://devdocs.magento.com/guides/v2.4/coding-standards/contributing.html)  page for more information.

Reviewer 2: Don't forget to include the tag, so name is not translated nor localized: [!DNL Upgrade Compatibility Tool].
So:

```
The [!DNL Upgrade Compatibility Tool] uses rules defined within our [Coding Standards](https://devdocs.magento.com/guides/v2.4/coding-standards/bk-coding-standards.html), to ensure your project is following Adobe Commerce best practices, and at the same time allowing you to improve and extend the [!DNL Upgrade Compatibility Tool] compatibility analysis by contributing to this project.

Refer to the [Contribute](https://devdocs.magento.com/guides/v2.4/coding-standards/contributing.html)  topic for more information on contributing to this project.
```

> ## Resources

Reviewer 2: Missing a couple of spaces in between title and lists:

```
## Resources

We have developed other resources to help you understand Adobe Commerce upgrades, these are links to key resources:

- [Upgrade guide](https://experienceleague.adobe.com/docs/commerce-operations/assets/adobe-commerce-2-4-upgrade-guide.pdf)
- [Upcoming releases](https://devdocs.magento.com/release/)
- [Community Resources](https://devdocs.magento.com/community/resources/resources.html) page for more information.

```

> We have developed other resources to help you understand Adobe Commerce upgrades, these are links to key resources:

Reviewer: ```suggestion

We have developed the following resources to help you understand Adobe Commerce upgrades:
```

> - [Upgrade guide](https://experienceleague.adobe.com/docs/commerce-operations/assets/adobe-commerce-2-4-upgrade-guide.pdf)

Reviewer: ```suggestion
- [Upgrade guide](https://experienceleague.adobe.com/docs/commerce-operations/upgrade-guide/overview.html)
```
